<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaDialog</class>
 <widget class="QDialog" name="MediaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>611</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="titleBar" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>979</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>650</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
   border : 1px  solid rgb(216, 216, 216); 
   border-bottom : none; 
   background-color: rgb(238, 238, 238);
   border-top-left-radius: 7px;
   border-top-right-radius: 7px;
}

/* 基本样式：透明无边框 */
QToolButton {
    border: none;                 /* 去除边框 */
    background-color: transparent; /* 背景透明 */
    padding:4 px;                 /* 适当内边距，避免内容太贴边 */
    margin: 0px;                  /* 去除外边距 */
}

/* 鼠标悬浮时：灰色背景 */
QToolButton:hover {
    background-color: rgba(200, 200, 200, 100); /* 半透明灰色 */
    border-radius: 3px;           /* 轻微圆角，视觉更友好 */
}

/* 可选：按下状态样式 */
QToolButton:pressed {
    background-color: rgba(150, 150, 150, 150); /* 按下时更深的灰色 */
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolButton" name="fixedToolButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>置顶</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/窗口置顶.svg</normaloff>:/a/icons/窗口置顶.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="qframe">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
        border-left: 1px solid rgb(188, 188, 188);  
        border-top: none;           /* 隐藏上边框 */
        border-right: 1px solid rgb(188, 188, 188) ;    
        border-bottom: none;        /* 隐藏下边框 */
        border-radius: none;
    }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="prevButton">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>上一张</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/icon.qrc">
           <normaloff>:/a/icons/上一张.svg</normaloff>:/a/icons/上一张.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="nextButton">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>下一张</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/icon.qrc">
           <normaloff>:/a/icons/下一张 .svg</normaloff>:/a/icons/下一张 .svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="thumbnailPreviewButton">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>预览</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/icon.qrc">
           <normaloff>:/a/icons/预览图片.png</normaloff>:/a/icons/预览图片.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomInButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>放大</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/放大.svg</normaloff>:/a/icons/放大.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomOutButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>缩小</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/缩小.svg</normaloff>:/a/icons/缩小.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="actualSizeOrfitWindowButton">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>图片原始大小</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/图片原始大小.svg</normaloff>:/a/icons/图片原始大小.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="qframe_2">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
        border-left: 1px solid rgb(188, 188, 188);  
        border-top: none;           /* 隐藏上边框 */
        border-right: 1px solid rgb(188, 188, 188) ;    
        border-bottom: none;        /* 隐藏下边框 */
        border-radius: none;
    }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="rotateCW90ToolButton">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>旋转</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/icon.qrc">
           <normaloff>:/a/icons/图片旋转-顺时针.svg</normaloff>:/a/icons/图片旋转-顺时针.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_16">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>编辑</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/icon.qrc">
           <normaloff>:/a/icons/图片编辑.svg</normaloff>:/a/icons/图片编辑.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_15">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>翻译</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/translate,翻译,翻译页面.svg</normaloff>:/a/icons/translate,翻译,翻译页面.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_8">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>提取文字</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/提取文字.svg</normaloff>:/a/icons/提取文字.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_9">
      <property name="minimumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>保存</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/下载.svg</normaloff>:/a/icons/下载.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="qframe_3">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>28</height>
       </size>
      </property>
      <property name="toolTip">
       <string>更多</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
        border-left: none;
        border-top: none;           /* 隐藏上边框 */
        border-right: 1px solid rgb(188, 188, 188) ;    
        border-bottom: none;        /* 隐藏下边框 */
        border-radius: none;
    }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>17</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton_17">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="toolTip">
          <string>更多</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/icon.qrc">
           <normaloff>:/a/icons/更多.svg</normaloff>:/a/icons/更多.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="minimizeToolButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>最小化</string>
      </property>
      <property name="styleSheet">
       <string notr="true">    border-radius: 0px;           /* 轻微圆角，视觉更友好 */
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/窗口最小化.svg</normaloff>:/a/icons/窗口最小化.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="maximizeToolButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>最大化</string>
      </property>
      <property name="styleSheet">
       <string notr="true">    border-radius: 0px;           /* 轻微圆角，视觉更友好 */
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/窗口最大化.svg</normaloff>:/a/icons/窗口最大化.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="closeToolButton">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolTip">
       <string>关闭</string>
      </property>
      <property name="styleSheet">
       <string notr="true">/* 鼠标悬浮时：灰色背景 */
QToolButton:hover {
    background-color: rgb(255, 65, 68); 
    border-radius: 0px;           /* 轻微圆角，视觉更友好 */
    border-top-right-radius: 7px;
}
/* 可选：按下状态样式 */
QToolButton:pressed {
    background-color: rgb(214, 55, 57); /* 按下时更深的灰色 */
}</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../res/icon.qrc">
        <normaloff>:/a/icons/窗口关闭.svg</normaloff>:/a/icons/窗口关闭.svg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../res/icon.qrc"/>
 </resources>
 <connections/>
</ui>
